apm-sniffer/apm-sdk-plugin/httpClient-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/httpClient/v4/HttpClientExecuteInterceptor.java

```diff
@@ -112,18 +115,14 @@ private boolean isUrl(String uri) {
     }
 
     private String buildSpanValue(HttpHost httpHost, String uri) {
-        if (isUrl(uri)) {
-            return uri;
-        } else {
-            StringBuilder buff = new StringBuilder();
-            buff.append(httpHost.getSchemeName().toLowerCase());
-            buff.append("://");
-            buff.append(httpHost.getHostName());
-            buff.append(":");
-            buff.append(port(httpHost));
-            buff.append(uri);
-            return buff.toString();
-        }
+        return isUrl(uri) ? uri : new StringBuilder()
```

Comment:
This is coding style. No one says this is better then if/else

Reply:
> > en, thanks for your reminder. this patch is not the code style at all.
> 
> I commented why I think it is a coding style thing.

ok, i see.